enterprise/backup/src/test/java/org/neo4j/backup/OnlineBackupCommandCcIT.java

```diff
@@ -137,6 +148,44 @@ public void backupCanNotBePerformedOverBackupProtocol() throws Exception
                         "--name=defaultport" ) );
     }
 
+    @Test
+    public void dataIsInAUsableStateAfterBackup() throws Exception
+    {
+        // given database exists
+        Cluster cluster = startCluster( recordFormat );
+
+        // and the database has indexes
+        createIndexes( cluster.getDbWithAnyRole( Role.LEADER ).database() );
+
+        // and the database is being populated
+        AtomicBoolean populateDatabaseFlag = new AtomicBoolean( true );
+        new Thread( () -> repeatedlyPopulateDatabase( clusterLeader( cluster ).database(), populateDatabaseFlag ) )
+                .start(); // populate db with number properties etc.
+        oneOffShutdownTasks.add( () -> populateDatabaseFlag.set( false ) ); // kill thread
+
+        // then backup is successful
+        String ip = TestHelpers.backupAddressCc( clusterLeader( cluster ).database() );
```

Comment:
Not really an ip. Perhaps address or hostnamePort?

Reply:
Addressed